deployments/hammerbox/environment.py

```diff
@@ -786,15 +786,12 @@ def env() -> Mapping[str, Optional[str]]:
         'AZUL_DUOS_SERVICE_URL': 'https://consent.dsde-prod.broadinstitute.org',
         'AZUL_TERRA_SERVICE_URL': 'https://firecloud-orchestration.dsde-prod.broadinstitute.org',
 
+        'AZUL_SHARE_ES_DOMAIN': '1',
+        'AZUL_ES_DOMAIN': 'azul-index-anvilprod',
+
         **(
             {
-                # $0.382/h × 2 × 24h/d × 30d/mo = $550.08/mo
-                'AZUL_ES_INSTANCE_TYPE': 'r6gd.xlarge.elasticsearch',
-                'AZUL_ES_INSTANCE_COUNT': '4',
             } if is_sandbox else {
-                # Personal deployments share an ES domain with `hammerbox`
-                'AZUL_SHARE_ES_DOMAIN': '1',
-                'AZUL_ES_DOMAIN': 'azul-index-hammerbox',
                 # Personal deployments use fewer Lambda invocations in parallel.
                 'AZUL_CONTRIBUTION_CONCURRENCY': '8',
                 'AZUL_AGGREGATION_CONCURRENCY': '8',
```

Comment:
It may be appropriate to simply drop this whole conditional section. If this is neither a personal deployment (with restricted concurrency) or a deployment managing an ES instance then it seem like just unused boilerplate. And also, the empty `{}` in `{} if is_sandbox else {` strikes me as odd, confirming that perhaps that whole block (L792-799) is indeed unnecessary.
```suggestion
```

Reply:
It was decided with @hannes-ucsc in PL to keep the `is_sandbox` variable and the personal deployment template-nature of this file at this time.
As such, using the ternary operator with an empty statement body `{ } if is_sandbox else { … }` is the most clean in this case (and consistent with other `environment.py` files).

Reply:
Roger that, in that case I retract my requested changes.